packages/avatar-group/src/vaadin-avatar-group-mixin.js

```diff
@@ -156,15 +122,52 @@ export const AvatarGroupMixin = (superClass) =>
 
     static get observers() {
       return [
-        '__i18nItemsChanged(i18n, items)',
+        '__i18nItemsChanged(__effectiveI18n, items)',
         '__openedChanged(_opened, _overflow)',
         '__updateAvatarsTheme(_overflow, _avatars, _theme)',
-        '__updateAvatars(items, __itemsInView, maxItemsVisible, _overflow, i18n)',
+        '__updateAvatars(items, __itemsInView, maxItemsVisible, _overflow, __effectiveI18n)',
         '__updateOverflowAvatar(_overflow, items, __itemsInView, maxItemsVisible)',
         '__updateOverflowTooltip(_overflowTooltip, items, __itemsInView, maxItemsVisible)',
       ];
     }
 
+    /**
+     * The object used to localize this component. To change the default
+     * localization, replace this with an object that provides all properties, or
+     * just the individual properties you want to change.
+     *
+     * The object has the following JSON structure and default values:
+     * ```
+     * {
+     *   // Translation of the anonymous user avatar tooltip.
+     *   anonymous: 'anonymous',
+     *   // Translation of the avatar group accessible label.
+     *   // {count} is replaced with the actual count of users.
+     *   activeUsers: {
+     *     one: 'Currently one active user',
+     *     many: 'Currently {count} active users'
+     *   },
+     *   // Screen reader announcement when user joins group.
+     *   // {user} is replaced with the name or abbreviation.
+     *   // When neither is set, "anonymous" is used instead.
+     *   joined: '{user} joined',
+     *   // Screen reader announcement when user leaves group.
+     *   // {user} is replaced with the name or abbreviation.
+     *   // When neither is set, "anonymous" is used instead.
+     *   left: '{user} left'
+     * }
+     * ```
+     * @type {!AvatarGroupI18n}
+     * @default {English/US}
+     */
+    get i18n() {
```

Comment:
One downside here is that Polymer analyzer will not be able to detect the default value anymore. The only way to fix this I found was this:
```
    /** @public ...i18n blurb... */
    this.i18n = { ...repeat all defaults again... }
```
Probably not worth it, since we explain the defaults in the comment itself.

Reply:
You can change `@type {!AvatarGroupI18n}` to `@return {!AvatarGroupI18n}` so that the `?` would be replaced with the property type as it was before. As for default value, I agree it's not necessary to have it there.

Reply:
Thanks, that looks better.